src/global/css.js

```diff
@@ -1,62 +1,53 @@
 import { doc } from '../config/environment';
-import noop from '../utils/noop';
-
-var css,
-	update,
-	styleElement,
-	head,
-	styleSheet,
-	inDom,
-	prefix = '/* Ractive.js component styles */\n',
-	styles = [],
-	dirty = false;
-
-if ( !doc ) {
-	// TODO handle encapsulated CSS in server-rendered HTML!
-	css = {
-		add: noop,
-		apply: noop
-	};
-} else {
+
+const PREFIX = '/* Ractive.js component styles */\n';
+
+// Holds current definitions of styles
+let styleDefinitions = [];
+
+// Flag to tell if we need to update the CSS
+let isDirty = false;
+
+// These only make sense on the browser. See additional setup below.
+let styleElement = null;
+let styleSheet = null;
+let styleProperty = null;
+
+function add( styleDefinition ) {
+	styleDefinitions.push( styleDefinition );
+	isDirty = true;
+}
+
+function toCSS() {
+	return PREFIX + styleDefinitions.map( s => `\n/* {${s.id}} */\n${s.styles}` ).join( '\n' );
```

Comment:
This implementation of `toCSS` (as a prototype method) doesn't seem to make sense when this `toCSS` returns all the styles defined. So...

- Either make `toCSS` a static function that returns all CSS defined in Ractive.
- Gather all the IDs from the instance down to the descendants, and grab only their CSS.